helpers/exec.go

```diff
@@ -112,6 +112,12 @@ func Kubectl(command string) (string, error) {
 	return RunProc(cmd, currentdir, false)
 }
 
+func KubectlWithProgress(ui *ui.UI, command string) (string, error) {
+	s := ui.Progressf(" Executing: kubectl %s", command)
```

Comment:
Are you sure the kubectl command is printed out only with `verbosity>0` ?

Reply:
that's handled automatically by that Progress function